### PR TITLE
feat(windows-agent): The agent creates the registry key if it does not exist

### PR DESCRIPTION
**Pull request by EduardGomezEscandell**
UDENG-2329

### CODE REVIEW

File: windows-agent/internal/proservices/registrywatcher/registry/registry_windows.go [hunk 1]
```diff
@@ -15,8 +15,8 @@ import (
 type Windows struct{}
 
 // HKCUOpenKey opens a key in the specified path under the HK_CURRENT_USER registry with read permissions.
-func (Windows) HKCUOpenKey(path string) (Key, error) {
-	key, err := registry.OpenKey(registry.CURRENT_USER, path, registry.READ)
+func (r Windows) HKCUOpenKey(path string) (Key, error) {
```
Comment (EduardGomezEscandell) [inline windows-agent/internal/proservices/registrywatcher/registry/registry_windows.go:18]:
```suggestion
func (Windows) HKCUOpenKey(path string) (Key, error) {
```

Unnecessary change

**Review by CarlosNihelton — APPROVED** [review]
LGTM
